app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,31 @@
 <!DOCTYPE html>
```

Comment:
- [ ] In order to be able to navigate all the project views without having to resort to the address bar of the browser, please, kindly add links:

- Clicking on each user takes you to their show page as shown in the user show wireframe.
- "See all Post" button opens the user's posts index as displayed on the posts wireframe.
- Clicking on a post on the user's posts index goes to the post details page and displays the post with all comments listed as shown on the post details wireframe.

## Explanation

We need to be able to navigate the project, without resorting to the address bar.

## Hints

In your views, in order to create links make use of the Rails methods `link_to` and `button_to`, `link_to image_tag` like these examples:

```ruby
link_to "Improve Your Ruby Skills", "/ruby-book" %>
button_to "Improve Your Ruby Skills", "/ruby-book" %>
link_to image_tag("path/to/image.png"), "http://www.mysite.com/"
```

That way, you will be able to navigate your project 💪